README.md

```diff
@@ -218,9 +218,9 @@ if err != nil {
 
 # Closing pipes
 
-If you've dealt with files in Go before, you'll know that you need to _close_ the file once you've finished with it. Otherwise, the program will retain what's called a _file handle_ (the kernel data structure which represents an open file). There is a limit to the total number of open file handles for a given program, and for the system as a whole, so a program which leaks file handles will eventually crash, and will waste resources in the meantime.
+If you've dealt with files in Go before, you'll know that you need to _close_ the file once you've finished with it. Otherwise, the program will retain what's called a _file handle_ (the kernel data structure which represents an open file). There is a limit to the total number of open filehandles for a given program, and for the system as a whole, so a program which leaks filehandles will eventually crash and will waste resources in the meantime.
```

Comment:
Stat. Google suggests that 'file handles' is about a thousand times more common than 'filehandles'.